plugins/uefi/fu-plugin-uefi.c

```diff
@@ -377,6 +377,31 @@ fu_plugin_uefi_update_splash (GError **error)
 	return fu_plugin_uefi_update_resource (re, 0, image_bmp, error);
 }
 
+static gboolean
+fu_plugin_uefi_esp_mounted (FuPlugin *plugin, GError **error)
+{
+	FuPluginData *data = fu_plugin_get_data (plugin);
+	g_autofree gchar *contents = NULL;
+	g_autofree gchar **lines = NULL;
```

Comment:
this only frees the `lines` pointer, you need to free the `lines` pointer, and all the pointers in `*lines`. The correct way to do this is the slightly clunky `g_auto(GStrv) lines = NULL`